opm-core/Installation.rst

```diff
@@ -239,13 +229,16 @@ User interface
 The default user interface is based on the web framework Mojolicious_. You need to install:
 
 * Perl (5.10 or above)
-* Mojolicious (4.63 or above, **less than 5.0**)
+* Mojolicious (4.63 or above, **less than 5.0 !**)
```

Comment:
I forgot to update it, but since opm 2.4, mojolicious 6.0+ is supported (I didn't check recently if a newer version break compatibility).  For now, just document "4.63 or above".

Reply:
I've tried with mojolicious 6.15 from xenial -> I had "render_not_found" errors when clicking on a server (no graph)
With mojolicious 7.21 from stretch or 7.55 from Cpan : it complains at opm start that the plugin "charset" is not there (and it seems that it was removed from mojo).
(with last opm-core version from github, of course)

For moment, I let this part.

Reply:
Agreed, I had a quick try with mojo 6.0 and also encountered some issue.  Which is weird since I did try this version back to https://github.com/OPMDG/opm-core/commit/10474f4cf3826e06d1d792e91878ad64787b6a9b